oal-parser/src/main/resources/code-templates/ServiceInstanceJVMGCDispatcherTemplate.ftl

```diff
@@ -54,6 +56,7 @@ public class ServiceInstanceJVMGCDispatcher implements SourceDispatcher<ServiceI
         }
         </#list>
     </#if>
+>>>>>>> ec8e105709a1377c62448cafe9049e79959827f1
```

Comment:
Solve the conflicts, please. And your pr in main repo, should include all new generated codes, which shouldn't pass ci based on this.